stock_picking_mixins/models/stock_picking_mixins.py

```diff
@@ -0,0 +1,37 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (http://www.gnu.org/licenses/lgpl-3.0.html)
+
+from odoo import fields, api, models
+
+
+class StockPickingMixins(models.AbstractModel):
+    _name = "stock.picking.state.mixins"
```

Comment:
This is lonely mixin, so should be like this:
```suggestion
class StockPickingMixin(models.AbstractModel):
    _name = "stock.picking.state.mixin"
```

- The addon contains one mixin for now, but it should allocate more models (mixins) in the future => mixins
- Every model should represent  a mixin  => mixin